### ISSUE TITLE
failure to list!

### ISSUE THREAD

**Issue by linuxcaffe** [blakesweeney/unite-taskwarrior#29]
Last night, ':Unite taskwarrior' was working great; generating a full tasklist.

This morning, not so much!

Now, while other commands are working; /context /project, /tags etc, but the basic listing command now simply opens an empty buffer, no list.

my Utw-related configs are here: https://gist.github.com/f0fc0edad098dba59c9b 

and I have no error messages to report!

**Comment by blakesweeney** [comment]
Huh. I'm not sure what's going on there. Could you uncomment lines 284 and 291 of autoload/unite/taskwarrior.vim?

**Comment by linuxcaffe** [comment]
lines uncommented, all I see is;

[unite.vim] Could not parse taskwarrior output

**Comment by linuxcaffe** [comment]
Found it! The error was connected to a context being set. Changing to "context none", and the list flows again!

**Comment by blakesweeney** [comment]
Are there no tasks in that context? It should probably tell you if there is nothing to list.

**Comment by linuxcaffe** [comment]
oh that's the thing, there ARE tasks in that context! I do believe 'tis a bug! See if you can reproduce, set a context with task, try listing with Utw..

**Comment by blakesweeney** [comment]
Yeah it seems to work for me, which is odd. Also the listing uses export which as far as I know isn't influenced by the context setting. Unless something has changed in newer versions. Plus, /tags and /projects use the same export mechanism.

**Comment by linuxcaffe** [comment]
Very odd. I'm using the latest task 2.4.5, latest develop branch of Utw, 

with tw context unset, tasklist, everything works.

tw context set, no tasklist, no /projects list, no /tags list.. weird!

**Comment by linuxcaffe** [comment]
this isn't necessarily a Utw problem, I'm seeing symptoms elsewhere, don't worry about it, for now :)

**Comment by blakesweeney** [comment]
Ok, then I'll close this issue for now. Reopen it if you find something specific to utw.

**Comment by linuxcaffe** [comment]
cool

**Comment by linuxcaffe** [comment]
alright, I found one taskwarrior context-related bug, and got a prompt fix, and that problem was solved.. but not this one!

Could it be that something in the task verbose output is messing with Utw?? The only remaining clue is vim message;

<pre>
"[unite.vim] Could not parse taskwarrior output   "
</pre>

and verbosity changes depending on context on/off.

also, because task v2.4.5 has seen a lot of fundamental changes recently, and because it is being prepared for beta to become v2.5 in the not-too-distant-future, I would recommend your working from the 2.4.5  (development) branch, and updating regularly, y'know, when you get a chance, unless you are already.. ;-)
thanks Blake!
